src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 200px;
+
+  &__imac {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  &__product {
+    &-name {
+      margin: 0 0 4px;
+
+      color: var(--text-standart-color);
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-code {
+      color: #616070;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      margin-bottom: 16px;
+    }
+
+    &-quality {
+      display: flex;
+      width: 100%;
+      justify-content: space-between;
+      align-items: flex-end;
+      margin-bottom: 24px;
+    }
+
+    &-reviews {
+      margin-left: auto;
+      color: var(--text-standart-color);
+      text-align: right;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 16px;
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 16px;
+
+      &-title {
+        font-size: 12px;
+        font-weight: 400;
+        color: #616070;
+        line-height: 18px;
+      }
+
+      &-cost {
+        margin-left: auto;
+        color: var(--text-standart-color);
+        text-align: right;
+        font-size: 16px;
+        font-weight: 700;
+        line-height: 18px;
+      }
+    }
+
+    &-count {
+      color: var(--text-standart-color);
+    }
+
+    &-button {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-sizing: border-box;
+      height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding: 11px 0;